stratum/hal/lib/barefoot/bfrt_table_manager.cc

```diff
@@ -444,8 +444,14 @@ ::util::StatusOr<::p4::v1::TableEntry> BfrtTableManager::BuildP4TableEntry(
             << ".";
     }
   }
+  
+  // Default actions do not have a priority, even when the table usually
+  // requires one. The SDE would return 0 (highest) which we must not translate.
+  if (request.is_default_action()) {
+    has_priority_field = false;
+  }
```

Comment:
I was thinking we can simplify to this

```c++
has_priority_field = !request.is_default_action();
```

However, the current one more clear to developers.
feel free to merge it when you think it Is ready